go/chi/Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM golang:1.11.1
+FROM golang:1.11.2
```

Comment:
Try the `1.11` version which always fetches the latest version like 1.11.2 now.

Reply:
I prefer be as strict as I can, since I use an external service to keep me update (and it's an easy way for me to learn 😁